README.md

```diff
@@ -18,17 +17,20 @@ as well as the
 [CloudEventConverters](https://github.com/googleapis/google-cloudevents-dotnet/blob/master/src/Google.Events/CloudEventConverters.cs)
 class that uses those attributes.
 
-The remaining packages each contain the Google event data types, but
-each package supports a different serialization framework. The
-intention is for frameworks such as the [.NET Functions
+The `Google.Events.Protobuf` contains the Google event data types,
+using Protocol Buffers as the serialization framework.
+
+The design allows for other serialization frameworks (e.g.
+System.Text.Json or Json.NET) to be supported in the future, each in
+a separate package. Frameworks such as the [.NET Functions
 Framework](https://github.com/GoogleCloudPlatform/functions-framework-dotnet)
-to depend only on `Google.Events`, but application code can depend on
-whichever package fits in best with their other dependencies. (If
+can depend only on `Google.Events`, but application code can depend
+on whichever package fits in best with their other dependencies. (If
 your application already depends on System.Text.Json, you may want
 to avoid a dependency on Google.Protobuf, or vice versa.)
```

Comment:
I would change the tense of this part:
```suggestion
can depend only on `Google.Events`, but application code could depend
on whichever package fits in best with their other dependencies. (If
your application already depended on System.Text.Json, you could be able
to avoid a dependency on Google.Protobuf, or vice versa.)
```

Reply:
Hmm... that feels a little more awkward to me. Will merge as-is, and we can tweak later. (We may well need to rewrite this entirely of course...)